ol2/tt_top/build.py

```diff
@@ -202,11 +204,20 @@ def pdn_align(x):
 	})
 
 	# Run flow
-	flow = TopFlow(
-		flow_cfg,
-		design_dir = ".",
-		pdk_root   = PDK_ROOT,
-		pdk        = "sky130A",
-	)
-
-	flow.start()
+
+	if args.open_in_klayout:
+		flow = OpenInKLayout(
```

Comment:
If all the args are the same and this is just a sub class of TopFlow then something like the following would avoid the duplication in case args change.

```python
flow_kls = OpenInKLayout if args.open_in_klayout else TopFlow
flow = flow_kls(
...
)
```

Reply:
Done!